README.md

```diff
@@ -10,40 +10,48 @@ This package is a thin wrapper around [TinyMCE](https://github.com/tinymce/tinym
 * For the TinyMCE Svelte Technical Reference, see: [TinyMCE Documentation - TinyMCE Svelte Technical Reference](https://www.tiny.cloud/docs/tinymce/7/svelte-ref/).
 * For our quick demos, check out the TinyMCE Svelte [Storybook](https://tinymce.github.io/tinymce-svelte/).
 
-
 ## Quick start
 
-### Create a Svelte App from a template
+### Create a Svelte App with Vite
 
-```
-npx degit sveltejs/template my-tiny-app
-cd my-tiny-app
+```bash
+npm create vite@5 tinymce-svelte-demo -- --template svelte
```

Comment:
- Using Vite@5 here instead of vite@latest to ensure that major updates to Vite do not break these instructions.
- Template used is `svelte` for simplicity. Could also use `svelte-ts` or [SvelteKit](https://kit.svelte.dev/docs/creating-a-project) here.